Gordon360/Services/AccountService.cs

```diff
@@ -55,7 +55,7 @@ public IEnumerable<AccountViewModel> GetAll()
         /// Fetches the account record with the specified email.
         /// </summary>
         /// <param name="email">The email address associated with the account.</param>
-        /// <returns>the student account information</returns>
+        /// <returns>the first account object which matches the email</returns>
         public AccountViewModel GetAccountByEmail(string email)
         {
             var account = _context.ACCOUNT.FirstOrDefault(x => x.email == email);
```

Comment:
```suggestion
            var account = _context.ACCOUNT.FirstOrDefault(x => x.email.ToLower().Equals(email.ToLower()));
```

Reply:
Why are you suggesting this change? We use `==` to compare strings throughout the codebase, and it's considered idiomatic C#.

Reply:
I figured you'd say that.

Reply:
You figured correctly 😄 

I'm all for making this change if there's a compelling reason to. But as I see it, the compelling reasons to use `==` are:
1. It is already consistently used throughout the codebase
2. It is more concise, making it easier to type.
3. It is easier to read because it is more familiar, because there is space between the operands, and because it is visually distinguishable from any regular function call.

I think the third point is especially relevant - when we use `String.Equals`, the reader has to actually read the function name to understand what's happening, because it could be any number of function calls. When we use `==`, it is not only shorter, but it takes less effort to understand.

Reply:
I found a compelling reason to use String.Equals - more performant case-insensitive matching.

That is, instead of:
```csharp
string1.ToLower() == string2.ToLower()
```
which actually performs 3 string operations and creates two more intermediate strings, we can use:
```csharp
string1.Equals(string2, StringComparison.OrdinalIgnoreCase)
```

I don't personally feel that this is better for readability, but it is definitively more performant, and it's pretty clear what's happening and the intent behind the code.
For readability's sake, we could consider a String extension method `EqualsIgnoreCase`, which just uses the StringComparions.OrdinalIgnoreCase comparison internally.

This is a pretty common pattern in our codebase so it might be worth considering.